partner_industry_secondary/README.rst

```diff
@@ -28,7 +28,7 @@ Partner Industry Secondary
 
 |badge1| |badge2| |badge3| |badge4| |badge5|
 
-This module adds secondary industries for partners.
+This module adds secondary industries for partners and allows to create a hierarchy of industries.
```

Comment:
This module adds a second field "Secondary Industries" behind the Odoo core one "Main Industry". Also this module allow hierarchy of industries.

![imagen](https://github.com/user-attachments/assets/91c407dc-0865-474d-a2cf-609030121163)